pcp/Platform.c

```diff
@@ -349,7 +349,14 @@ bool Platform_init(void) {
 
    if (opts.context == PM_CONTEXT_ARCHIVE) {
       gettimeofday(&pcp->offset, NULL);
+#if PMAPI_VERSION >= 3
+    {
```

Comment:
We wont need these extra braces @kmcdonell - htop code base uses c99 by default and declaring variables in-place like this (without restricting scope) is the normal htop style.